readme.md

```diff
@@ -205,7 +205,7 @@ If your Nextcloud is running and you are logged in as admin in your Nextcloud, y
 ### How to change the domain?
 **⚠️ Please note:** Editing the configuration.json manually and making a mistake may break your instance so please create a backup first!
 
-If you set up a new AIO instance, you need to enter a domain. Currently there is no way to change this domain afterwards from the AIO interface. So in order to change it, you need to edit the configuration.json manually that is most likely stored in `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json`, subsitute each occurrence of your old domain with your new domain and save and write out the file. Afterwards restart your containers from the AIO interface and everything should work as expected if the new domain is correctly configured.<br>
+If you set up a new AIO instance, you need to enter a domain. Currently there is no way to change this domain afterwards from the AIO interface. So in order to change it, you need to edit the configuration.json manually that is most likely stored in `nextcloud-aio-mastercontainer:/mnt/docker-aio-config/data/configuration.json`, subsitute each occurrence of your old domain with your new domain and save and write out the file. Afterwards restart your containers from the AIO interface and everything should work as expected if the new domain is correctly configured.<br>
```

Comment:
I'd say lets keep the old path as well...

Reply:
maybe there is a way to launch an alpine container for this temporarily which includes nano or something that could be used for editing?

Reply:
maybe `sh -c "apk add nano && nano ..."`